plyer/facades/screenshot.py

```diff
@@ -0,0 +1,60 @@
+'''
+Screenshot
+=====
+
+The :class:`Screenshot` is used for recording audio.
+
+Default path for taking screenshot is set in platform implementation.
+
+Simple Examples
+---------------
+
+To get the file path::
+
+    >>> screenshot.file_path
+    '/sdcard/test.jpg'
+
+To set the file path::
+
+    >>> import os
+    >>> current_list = os.listdir('.')
+    ['/sdcard/testrecorder.jpg', '/sdcard/testrecorder1.jpg',
+    '/sdcard/testrecorder2.jpg', '/sdcard/testrecorder3.jpg']
+    >>> file_path = current_list[2]
+    >>> screenshot.file_path = file_path
+
+To take screenshot::
+
+    >>> from plyer import screenshot
+    >>> screenshot.take_shot()
+'''
+
+
+class Screenshot(object):
+    '''
+    ScreenShot facade.
+    '''
+    _file_path = ''
+
+    def __init__(self, file_path):
+        super(Screenshot, self).__init__()
+        self._file_path = file_path
+
+    def take_shot(self):
+        self._take_shot()
+
+    @property

```

Comment:
See [how](http://stackoverflow.com/q/64749/5994041) to remove `^M` character in your code.